std/range/package.d

```diff
@@ -16,55 +16,38 @@ $(I On Iteration)).
 
 Submodules:
 
-This module has a few submodules:
+This module has a two submodules:
 
+$(LIST
+$(DIV ,
 The $(LINK2 std_range_primitives.html, $(D std._range.primitives)) submodule
 provides basic _range functionality. It defines several templates for testing
 whether a given object is a _range, what kind of _range it is, and provides
 some common _range operations.
-
+),
+$(DIV ,
 The $(LINK2 std_range_interfaces.html, $(D std._range.interfaces)) submodule
 provides object-based interfaces for working with ranges via runtime
 polymorphism.
+))
 
 The remainder of this module provides a rich set of _range creation and
 composition templates that let you construct new ranges out of existing ranges:
 
 $(BOOKTABLE ,
-    $(TR $(TD $(D $(LREF retro)))
-        $(TD Iterates a bidirectional _range backwards.
-    ))
-    $(TR $(TD $(D $(LREF stride)))
-        $(TD Iterates a _range with stride $(I n).
+    $(TR $(TD $(D $(LREF NullSink)))
```

Comment:
I don't think this should be at the top; the sort order should be case-insensitive.